<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FriendListWgt</class>
 <widget class="QWidget" name="FriendListWgt">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>288</width>
    <height>679</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FriendListWgt</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QTreeView::branch:closed:has-children:has-siblings,QTreeView::branch:closed:!has-children:!has-siblings{border-image: none; image: url(:/LoginWgt/images/mainwindow/arrow_r.png);}



QTreeView::branch:open:has-children:!has-siblings,QTreeView::branch:open:has-children:has-siblings{border-image: none; image: url(:/LoginWgt/images/mainwindow/arrow_d.png);}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>150</height>
      </size>
     </property>
     <widget class="QLineEdit" name="lineEdit_2">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>60</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>40</y>
        <width>54</width>
        <height>12</height>
       </rect>
      </property>
      <property name="text">
       <string>昵称</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>40</y>
        <width>31</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>等级</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>40</y>
        <width>21</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>vip</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>40</y>
        <width>61</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>头像</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>50</y>
        <width>51</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>天气</string>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>0</y>
        <width>226</width>
        <height>32</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QToolButton" name="closeBtn_3">
         <property name="minimumSize">
          <size>
           <width>32</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>32</width>
           <height>30</height>
          </size>
         </property>
         <property name="text">
          <string>空间</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="closeBtn_2">
         <property name="minimumSize">
          <size>
           <width>32</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>32</width>
           <height>30</height>
          </size>
         </property>
         <property name="text">
          <string>皮肤</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="minSizeBtn_3">
         <property name="minimumSize">
          <size>
           <width>32</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>32</width>
           <height>30</height>
          </size>
         </property>
         <property name="text">
          <string>勋章</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="minSizeBtn_4">
         <property name="minimumSize">
          <size>
           <width>32</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>32</width>
           <height>30</height>
          </size>
         </property>
         <property name="text">
          <string>邮箱</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="minSizeBtn_2">
         <property name="minimumSize">
          <size>
           <width>32</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>32</width>
           <height>30</height>
          </size>
         </property>
         <property name="text">
          <string>设置</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="minSizeBtn">
         <property name="minimumSize">
          <size>
           <width>32</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>32</width>
           <height>30</height>
          </size>
         </property>
         <property name="text">
          <string>最小</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="closeBtn">
         <property name="minimumSize">
          <size>
           <width>32</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>32</width>
           <height>30</height>
          </size>
         </property>
         <property name="text">
          <string>关闭</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>99</y>
        <width>261</width>
        <height>41</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QLineEdit" name="searchLineEdit"/>
       </item>
       <item>
        <widget class="QToolButton" name="toolButton">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QPushButton" name="msgListBtn">
       <property name="minimumSize">
        <size>
         <width>90</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>消息</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="friendListBtn">
       <property name="minimumSize">
        <size>
         <width>90</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>联系人</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="spaceListBtn">
       <property name="minimumSize">
        <size>
         <width>90</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>空间</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QStackedWidget" name="mainStackedWgt">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="page">
      <widget class="QPushButton" name="pushButton_8">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>160</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>1</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="page_2">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QPushButton" name="pushButton_9">
           <property name="minimumSize">
            <size>
             <width>32</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>32</width>
             <height>30</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_11">
           <property name="minimumSize">
            <size>
             <width>32</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>32</width>
             <height>30</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QWidget" name="widget_3" native="true">
         <property name="minimumSize">
          <size>
           <width>270</width>
           <height>0</height>
          </size>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QWidget" name="widget_2" native="true">
            <widget class="QPopupWgt" name="popUpWgt" native="true">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>0</y>
               <width>270</width>
               <height>25</height>
              </rect>
             </property>
             <property name="minimumSize">
              <size>
               <width>270</width>
               <height>25</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>25</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true"/>
             </property>
            </widget>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_3">
      <widget class="QPushButton" name="pushButton_10">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>210</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>3</string>
       </property>
      </widget>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QPushButton" name="pushButton_4">
       <property name="minimumSize">
        <size>
         <width>32</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>32</width>
         <height>30</height>
        </size>
       </property>
       <property name="text">
        <string>PushButton</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_5">
       <property name="minimumSize">
        <size>
         <width>32</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>32</width>
         <height>30</height>
        </size>
       </property>
       <property name="text">
        <string>PushButton</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_6">
       <property name="minimumSize">
        <size>
         <width>32</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>32</width>
         <height>30</height>
        </size>
       </property>
       <property name="text">
        <string>PushButton</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_7">
       <property name="minimumSize">
        <size>
         <width>32</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>32</width>
         <height>30</height>
        </size>
       </property>
       <property name="text">
        <string>PushButton</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QPopupWgt</class>
   <extends>QWidget</extends>
   <header>QPopupWgt.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
